### PR TITLE
[FIX] OpenSWATH: OSW transition-level reports without MI scores

### PR DESCRIPTION
**Pull request by grosenberger**
This PR fixes a bug in the OSW export functionality if transition-level scoring is enabled but MI scores are not used.

### CODE REVIEW

File: src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h [hunk 1]
```diff
@@ -392,23 +392,50 @@ namespace OpenMS
...
+            for (int i = 0; i < id_target_num_transitions; ++i)
+            {
+              if (id_target_total_mi[i] == "") {
+                id_target_total_mi[i] = "NULL";
+              }
+              if (id_target_ind_mi_score[i] == "") {
+                id_target_ind_mi_score[i] = "NULL";
+              }
+              if (id_target_ind_mi_ratio_score[i] == "") {
+                id_target_ind_mi_ratio_score[i] = "NULL";
```
Comment (hroest) [inline src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h]: why do you expect empty strings? does OpenSWATH write out empty strings?

Comment (hroest) [reply src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h]: PS: also can you change the style to
```
if ... 
{
}
```

**Comment by hroest** [comment]
looks good, some minor questions

File: src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h (hunk 1 above)
Comment (grosenberger) [reply src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h]: The transition-level scores are still reported as a concatenated meta feature (e.g. "2;3;4"). If (some of) the MI scores are disabled, an empty string might be reported. This part might be worthwhile to refactor in the future.

I changed the style accordingly.

**Pull request merged**